client/my-sites/activity/activity-log-v2/index.tsx

```diff
@@ -45,18 +89,12 @@ const ActivityLogV2: FunctionComponent = () => {
 			} ) }
 			wideLayout={ ! isJetpackCloud() }
 		>
+			{ siteId && <QuerySitePlans siteId={ siteId } /> }
```

Comment:
I think we need to add a `QuerySitePurchases` component, so we can know if the site has a Backup product. Otherwise, `siteHasBackupProductPurchase` return false even if the site has Jetpack Backup.

Reply:
Good catch! I added that component and it did fix the issue.